clang/include/clang/CIR/Dialect/IR/CIRDialect.td

```diff
@@ -44,6 +44,7 @@ def CIR_Dialect : Dialect {
     static llvm::StringRef getModuleLevelAsmAttrName() { return "cir.module_asm"; }
     static llvm::StringRef getGlobalCtorsAttrName() { return "cir.global_ctors"; }
     static llvm::StringRef getGlobalDtorsAttrName() { return "cir.global_dtors"; }
+    static llvm::StringRef getExceptionAttrName() { return "exception"; }
```

Comment:
I don't really like this name. I think "mayThrow" would be better, and it aligns with LLVM IR function attribute flag of the same name.